### PR TITLE
Add http request qa 358

### PR DESCRIPTION
**Pull request by chris-faulkner-mojdt**
Added swagger and some .rest files to exercise the endpoints

### CODE REVIEW

File: src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                        regex("(\\/info)"),
+                        regex("(\\/health)"),
+                        regex("(\\/case/.*)"),
+                        regex("(\\/offender/.*)"),
+                        regex("(\\/offenders/.*)"),
+                        regex("(\\/court/.*)"))))
+                .build();
+
+        docket.genericModelSubstitutes(Optional.class);
+        docket.directModelSubstitute(ZonedDateTime.class, java.util.Date.class);
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java]: What's the reason for the substitutions here?

File: src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            properties.put("version", "?");
+            return new BuildProperties(properties);
+        }
+    }
+
+    private Contact contactInfo() {
+        return new Contact(
+                "HMPPS Probation in Court Team",
+                "",
+                "@digital.justice.gov.uk");
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java]: Might as well provide some actual contact info, happy to have my address down as I'm a permie. `john.evans+pictsupport@digital.justice.gov.uk`

Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java]: Cheers - wondered about that

Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/application/SwaggerConfig.java]: yes

**Pull request merged**